tango_simlib/utilities/helper_module.py

```diff
@@ -81,6 +81,17 @@ def get_server_name():
     server_name = executable_name + '/' + sys.argv[1]
     return server_name
 
+def get_file_name():
+    """Gets the db file name from the command line arguments
+
+    Returns
+    -------
+    file_name : str
+        file used as tango database
+    """
+    file_name = sys.argv[4].split('=')[1]
```

Comment:
Is it guaranteed that anyone using tango simlib to launch a device server will provide this many arguments?  What happens if they provide them in a different order?  The `-file` argument is not positional.

Reply:
Sure.